website/public/css/global-modules.styl

```diff
@@ -68,6 +68,8 @@ hrpg-button-color-mixin($hrpg-button-color, highlight=false)
       color: darken($hrpg-button-color, 70%) !important
   .glyphicon
     color:darken($hrpg-button-color, 60%)
+  .tooltip
+    border:0
```

Comment:
This was necessary, because the tooltips looked like this:

![image](https://cloud.githubusercontent.com/assets/3625477/9097190/4d2e43c0-3bc2-11e5-9f16-cd93ff8a57c4.png)

I have a feeling this might be a bit hacky, though. Suggestions for improvement?